### PR TITLE
TRIVIAL: Update translations

### PR DESCRIPTION
**Pull request by patrikbraborec**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6250/ : SUCCESS in 17m 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/1937/ : SUCCESS in 5s

File: src/internal/translations/de-DE.json [hunk 1]
```diff
@@ -63,8 +69,8 @@
     "dashboard.bucket.measures_title.table": "Messungen",
     "dashboard.bucket.attribute_title.table": "Zeilen",
     "dashboard.bucket.columns_title.table": "Spalten",
-    "dashboard.error.missing_primary_bucket_item.heading": "Kein primäres Maß in Ihrer Betrachtung",
-    "dashboard.error.missing_primary_bucket_item.text": "Fügen Sie Ihrer Ansicht ein primäres Maß hinzu oder schalten Sie auf Tabelle um.\nWenn Sie fertig sind, können Sie es speichern.",
+    "dashboard.headline.error.missing_bucket_item.heading": "Kein primäres Maß in Ihrer Betrachtung",
```
Comment (petrjanu) [inline src/internal/translations/de-DE.json]: This is weird. If someone renamed the key, he should rename it also in translations. Please find out who renamed the key and why the did not do it also in translations. This should not happen - author should know about the risks of this approach.

Comment (petrjanu) [reply src/internal/translations/de-DE.json]: But if it is renamed in en-US, we have to merge it asap...

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

File: src/internal/translations/de-DE.json (hunk 1 above)
Comment (patrikbraborec) [reply src/internal/translations/de-DE.json]: Done.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3247/ : SUCCESS in 17m 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6263/ : SUCCESS in 10m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1521/ : SUCCESS in 34s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2140/ : SUCCESS in 21m 24s

**Pull request merged**